ui/app/src/state/actions/preview.ts

```diff
@@ -144,11 +157,18 @@ export const instanceDragEnded = /*#__PURE__*/ createAction('INSTANCE_DRAG_ENDED
 export const navigationRequest = /*#__PURE__*/ createAction('NAVIGATION_REQUEST');
 export const reloadRequest = /*#__PURE__*/ createAction('RELOAD_REQUEST');
 export const desktopAssetDrop = /*#__PURE__*/ createAction('DESKTOP_ASSET_DROP');
-export const desktopAssetUploadComplete = /*#__PURE__*/ createAction<{ record; path: string }>(
+export const desktopAssetUploadComplete = /*#__PURE__*/ createAction<{ record: ElementRecord; path: string }>(
```

Comment:
This couldn't/shouldn't be an element record. An element can't be posted through windows. Double check

Reply:
Per discussion I moved the desktopAssetUpload actions to guest